packer/README.md

```diff
@@ -0,0 +1,57 @@
+
+# CommandoVM & HashiCorp's Packer
+
+Welcome to the packer builder of CommandoVM - a fully automated installation of the customizable commandvm platform.
+
+## Requirements
+
+* a hypervisor ( below are current hypervisors )
+  * [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+* HashiCorp's [packer](https://www.packer.io/downloads)
+* [packer provisioner windows update](https://github.com/rgl/packer-provisioner-windows-update)
+* 60 GB Hard Drive
+* 2 GB RAM
+
+## Recommended
+
+* 80+ GB Hard Drive
+* 4+ GB RAM
```

Comment:
specifically for this section, I was matching what is on the README file for CommandVM, under their [requirements](https://github.com/fireeye/commando-vm#requirements)... ,but I think those requirements are for the VM. Which we would want to change this to be machine requirements ( which would be more probably 4 required and 8+ recommended or something like that ). So, I am curious as to what you think of this.